tika-parsers/src/main/resources/org/apache/tika/parser/captioning/tf/model_info.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<service name="im2txt" port="8764">
+    <model_main>
+        <checkpoint_path>/usr/local/models/model_main/model.ckpt-1000000</checkpoint_path>
+        <vocab_file>/usr/local/models/model_main/word_counts.txt</vocab_file>
```

Comment:
My opinion: Yes place all of them under a directory that says `tika`.
 Since the docker image is ubuntu, `/usr/share/apache-tika/`  seems the right place.
Place your python code under `/usr/share/apache-tika/src/dl/image/caption`
Place your models under `/usr/share/apache-tika/models/dl/image/caption`
Then setup PATH and PYTHONPATH variables correctly


@chrismattmann thoughts on this?